xml/System.Data.Common/DbProviderFactory.xml

```diff
@@ -96,8 +96,10 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Specifies whether the specific <see cref="T:System.Data.Common.DbProviderFactory" /> supports the <see cref="T:System.Data.Common.DbCommandBuilder" /> class.</summary>
+        <value>
+          <see langword="true" /> if the instance of the <see cref="T:System.Data.Common.DbProviderFactory" /> supports the <see cref="T:System.Data.Common.DbCommandBuilder" /> class; otherwise <see langword="false" />.
```

Comment:
```suggestion
          <see langword="true" /> if this instance supports the <see cref="T:System.Data.Common.DbCommandBuilder" /> class; otherwise <see langword="false" />.
```